src/Sylius/Bundle/ChannelBundle/Tests/DependencyInjection/Compiler/CompositeRequestResolverPassTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ChannelBundle\Tests\DependencyInjection\Compiler;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\DefinitionHasMethodCallConstraint;
+use Sylius\Bundle\ChannelBundle\DependencyInjection\Compiler\CompositeRequestResolverPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class CompositeRequestResolverPassTest extends AbstractCompilerPassTestCase
+{
+    /**
+     * @test
+     */
+    public function it_collects_tagged_request_based_channel_contexts()
```

Comment:
You are right, but on the other hand shouldn't we keep consistency? At the end tests in theme bundle or api tests are written with camel case. Should we force some convention?